### ISSUE TITLE
Scrollspy Not Adding 'Current' for Screen Readers to Active Anchors

### ISSUE THREAD

**Issue by chrisbo** [twbs/bootstrap#25869]
Shouldn't Scrollspy be adding `<span class="sr-only">(current)</span>` to active anchors too, or am I missing the point or something?

**Comment by patrickhlauke** [comment]
kind of debatable, as to hear that announcement AT users would navigate back/set focus to the anchor, thus likely changing the scroll position

**Comment by patrickhlauke** [comment]
unless they use additional features like bringing up a link list, in which case the "current" text would be included correctly. seems a bit of an edge case, to be honest (and of low utility to AT users)

**Comment by chrisbo** [comment]
Thanks for your input on this. So I guess the best thing on a one-page site is to just add current as normal - even if it is also an anchor e.g.

```
<a class="nav-item nav-link" href="#home">Home <span class="sr-only">(current)</span></a>
<a class="nav-item nav-link" href="#products">Products</a>
<a class="nav-item nav-link" href="#services">Services</a>
<a class="nav-item nav-link active" href="#contact">Contact</a>
```

**Comment by patrickhlauke** [comment]
if these are all on the single page, there's no point in adding the "current" thing at all

**Comment by patrickhlauke** [comment]
going to close this, considering my last comment. the addition of "current" doesn't really make sense for in-page anchor links.
